hopsanc/include/hopsanc.h

```diff
@@ -0,0 +1,20 @@
+#ifndef HOPSANC_H
+#define HOPSANC_H
+
+#include "hopsanc_win32dll.h"
+
+extern "C" {
+    HOPSANC_DLLAPI int loadLibrary(const char* path);
+    HOPSANC_DLLAPI int loadModel(const char* path);
+    HOPSANC_DLLAPI int setParameter(const char* name, const char *value);
+    HOPSANC_DLLAPI int setStartTime(double value);
+    HOPSANC_DLLAPI int setTimeStep(double value);
+    HOPSANC_DLLAPI int setStopTime(double value);
+    HOPSANC_DLLAPI int setLogSamples(unsigned long int value);
+    HOPSANC_DLLAPI int simulate();
+    HOPSANC_DLLAPI double *getTimeVector(int &value);
```

Comment:
C does not have references, so this is in fact a C++ API with C-style name mangling
But change the references to pointers, then its valid C

Reply:
Already fixed, since the size argument was removed.